extensions/amp-animation/0.1/web-animations.js

```diff
@@ -792,16 +794,10 @@ export class MeasureScanner extends Scanner {
       '"iterationStart" is invalid: %s',
       newTiming.iterationStart
     );
-    user().assertEnumValue(
-      WebAnimationTimingDirection,
-      /** @type {string} */ (direction),
-      'direction'
-    );
-    user().assertEnumValue(
-      WebAnimationTimingFill,
-      /** @type {string} */ (fill),
-      'fill'
-    );
+
+    userAssertIsWebAnimationTimingDirection(direction);
```

Comment:
I moved several checks into the file that defines the enum, so that the transform can evaluate them.

If we'd like, we can ensure this usage with a [lint rule](https://astexplorer.net/#/gist/d2fb96484a5b5a7dbadc3e84f82cf9f9/e52accd3d636004d72496f2c6b3efbe85435afe0).